components/StyledSelect.js

```diff
@@ -115,7 +115,9 @@ export const makeStyledSelect = SelectComponent => styled(SelectComponent).attrs
     error,
     controlStyles,
     isSearchable,
+    menuPlacement,
   }) => ({
+    menuPlacement,
```

Comment:
The prop is much welcome, but we don't need to add it there. All props for `styled-select` are automatically passed down to `react-select`.

```suggestion
  }) => ({
```